pf/internal/check/check_test.go

```diff
@@ -96,6 +155,97 @@ func TestSensitiveIDWithOverride(t *testing.T) {
 	})
 }
 
+func TestInvalidInputID(t *testing.T) {
+	tests := []struct {
+		name            string
+		idSchema        idSchema
+		isInputProperty bool
+	}{
+		{name: "Required", idSchema: idSchema{required: true, optional: false, computed: false}, isInputProperty: true},
```

Comment:
Style nit: Go likes to skip zero value init like `optional:false`, though I can see that deviating from that style adds some clarity here.

Reply:
Yeah I intentionally specified it to make it explicitly clear what the cases where.